cpp/lib/ArrowChunkIterator.cpp

```diff
@@ -838,10 +838,12 @@ SF_STATUS STDCALL ArrowChunkIterator::getCellAsString(
         auto values = m_columns[colIdx].arrowBinary->GetValue(m_currRowIndexInBatch, &len);
         std::vector<char> buffer(len * 2 + 1);
         char* ptr = buffer.data();
+        size_t sizeLeft = buffer.size();
         for (int i = 0; i < len; i++)
         {
-            sprintf(ptr, "%02X", values[i]);
+            sb_sprintf(ptr, sizeLeft, "%02X", values[i]);
```

Comment:
assuming sb_sprintf available on all platforms.

Reply:
Yes, it's copy of simba_xxxx functions in libsnowflakeclient.